src/components/CompanionWindow.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import Divider from '@material-ui/core/Divider';
 import CloseIcon from '@material-ui/icons/Close';
+import OpenWith from '@material-ui/icons/OpenWith';
```

Comment:
Can we make this `OpenWithSharp` to match other Sharp icons now that we've standardized?

Reply:
Or can we use the drag icon listed in #1947 ?   DragIndicatorSharp

Reply:
Sure! I suppose I misunderstood 
> I don't think this icon is directly needed yet.

in https://github.com/ProjectMirador/mirador/pull/1967#issuecomment-467540566